plugins/easyimage/plugin.js

```diff
@@ -163,6 +168,45 @@
 				},
 
 				init: function() {
+					// Natural width of the image can be fetched only after image is loaded.
+					// However cached images won't fire `load` event, but just mark themselves
+					// as complete.
+					function getNaturalWidth( image, callback ) {
+						var $image = image.$;
+
+						if ( $image.complete && $image.naturalWidth ) {
+							return callback( $image.naturalWidth );
+						}
+
+						image.once( 'load', function() {
+							callback( $image.naturalWidth );
+						} );
+					}
+
+					function setImageWidth( widget, height ) {
```

Comment:
This is what I deliberately wanted to avoid. Plugin scope is getting super crowdy because of all the helper function, I find it pretty hard to find something useful there - so wanted to keep the functions closer to where they are used, and I think it makes more sense here.